core/network/src/test/mod.rs

```diff
@@ -610,10 +626,14 @@ pub trait TestNetFactory: Sized {
 
 	/// Create new test network with this many peers.
 	fn new(n: usize) -> Self {
+		trace!(target: "test_network", "Creating test network");
```

Comment:
Why all this spam?

Reply:
I used it for debugging BABE.  This branch is based on that one.